src/components/CoursesTable/CoursesTable.jsx

```diff
@@ -55,36 +35,26 @@ const CoursesTable = () => {
 
   return (
     <>
-      <table className="table table-hover">
-        <thead>
-          <tr>
-            <th scope="col">Course Name</th>
-            <th scope="col">Title</th>
-            <th scope="col">Description</th>
-            <th scope="col">Duration (Hrs) </th>
-            <th scope="col">Instructor</th>
-            <th scope="col">Date Added</th>
-            <th scope="col">Action</th>
-          </tr>
-        </thead>
-        <tbody>
+      <div className="container overflow-hidden text-center">
+        <div className="d-flex">
           {getDataByPage().map((course) => (
-            <tr className="table-primary" key={course.id}>
-              <td>{course.course_category}</td>
-              <td>
-                <Link to={`courses/${course.course_id}`}>{course.course_title}</Link>
-              </td>
-              <td>{course.course_description}</td>
-              <td>{course.course_duration}</td>
-              <td>{course.course_instructor}</td>
-              <td>{moment(course.date_added).format("LL")}</td>
-              <td>
-                <input type="submit" className="btn btn-secondary" value="View" style={{ textAlign: "center" }} />
-              </td>
-            </tr>
+            <div className="col" key={course.id}>
+              <div className="d-flex">
+                <SingleCourseCard
+                  courset={course.course_title}
+                  coursedes={course.course_description}
+                  courseid={course.course_id}
+                  coursecat={course.course_category}
+                  coursetime={course.course_duration}
+                  courseinstructor={course.course_instructor}
+                />
+              </div>
+
+              {/* {console.log(course.course_category)} */}
```

Comment:
remove commented out code to have a cleaner code base

Reply:
done